examples/custom_distribution.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<p>In <code>v4</code>  the <code>testval</code>  kwarg is deprecated in favor of <code>initval</code>.  Is it really necessary though? If so why?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
I don't think the `testval` is really serving any purpose. I think I just included it because it wasn't working without it.

Reply:
It turns out I needed the set a `testval` because I hadn't specified `self.mode` in the distribution. I'll remove it now that I'm specifying `self.mode`